templates/include/prism/ast.h.erb

```diff
@@ -157,11 +157,11 @@ typedef struct pm_node {
  *<%= line %>
 <%- end -%>
  *
- * Type: <%= node.type %>
-<%- if (node_flags = node.flags) -%>
- * Flags:
+ * Type: ::<%= node.type %>
+<% if (node_flags = node.flags) %>
+ * Applicable flags (#pm_<%= node_flags.human %>):
```

Comment:
```suggestion
 * Flags (#pm_<%= node_flags.human %>):
```

Reply:
Hmmm, are you sure? It seems out-of-context on its own, IMO.